### PR TITLE
HentaiDex | Added HentaiDex

### PR DESCRIPTION
**Pull request by KenjieDec**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/en/hentaidex/src/eu/kanade/tachiyomi/extension/en/hentaidex/HentaiDex.kt [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+import kotlin.concurrent.thread
+
+class HentaiDex : ParsedHttpSource() {
+
+    override val name = "HentaiDex"
```
Comment (AwkwardPeak7) [inline src/en/hentaidex/src/eu/kanade/tachiyomi/extension/en/hentaidex/HentaiDex.kt]: website calls itself DEXHentai

**Review by AwkwardPeak7 — CHANGES_REQUESTED** [review]
the site looks like `MangaThemesia` multisrc...?

File: src/en/hentaidex/src/eu/kanade/tachiyomi/extension/en/hentaidex/HentaiDex.kt (hunk 1 above)
Comment (KenjieDec) [reply src/en/hentaidex/src/eu/kanade/tachiyomi/extension/en/hentaidex/HentaiDex.kt]: It seems like the website also identifies as HentaiDex 
image

**Comment by KenjieDec** [comment]
> the site looks like `MangaThemesia` multisrc...?

Should I change to that multisrc?

**Comment by AwkwardPeak7** [comment]
> Should I change to that multisrc?

yes

File: src/en/hentaidex/build.gradle [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ext {
+    extName = 'HentaiDex'
+    extClass = '.HentaiDex'
+    themePkg = 'mangathemesia'
+    baseUrl = 'https://dexhentai.com'
+    overrideVersionCode  = 1
```
Comment (AwkwardPeak7) [inline src/en/hentaidex/build.gradle]:
```suggestion
    overrideVersionCode  = 0
```

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**